odl/operator/default_ops.py

```diff
@@ -75,21 +75,7 @@ def scalar(self):
         return self.__scalar
 
     def _call(self, x, out=None):
-        """Scale input and write to output.
-
-        Parameters
-        ----------
-        x : `domain` element
-            input vector to be scaled
-        out : `range` element, optional
-            Output vector to which the result is written
-
-        Returns
-        -------
-        out : `range` element
-            Result of the scaling. If ``out`` was provided, the
-            returned object is a reference to it.
-        """
+        """Scale ``x`` and write to ``out`` if given."""
```

Comment:
Now that there are no more examples in `_call` and it's not part of the online doc, I don't see a point in having the exact same docstring except for the first line everywhere. We have documentation on how to write `_call`, so the removed part didn't add any information.